src/util/misc/svgParsing.ts

```diff
@@ -139,20 +141,20 @@ export const matrixToSVG = (transform: TMat2D) =>
  * @param value
  * @returns
  */
-export const colorPropToSVG = (prop: string, value?: any) => {
+export const colorPropToSVG = (prop: string, value?: any, separator: ColorSeparator = ':') => {
   if (!value) {
-    return `${prop}: none; `;
+    return separator === ':' ? `${prop}: none; ` : `${prop}="none" `;
   } else if (value.toLive) {
-    return `${prop}: url(#SVGID_${value.id}); `;
+    return separator === ':' ? `${prop}: url(#SVGID_${value.id}); ` : `${prop}="url(#SVGID_${value.id})" `;
   } else {
     const color = new Color(value),
       opacity = color.getAlpha();
 
-    let str = `${prop}: ${color.toRgb()}; `;
+    let str = separator === ':' ? `${prop}: ${color.toRgb()}; ` : `${prop}="${color.toRgb()}" `;
 
     if (opacity !== 1) {
       //change the color in rgb + opacity
-      str += `${prop}-opacity: ${opacity.toString()}; `;
+      str += separator === ':' ? `${prop}-opacity: ${opacity.toString()}; ` : `${prop}-opacity="${opacity.toString()}" `;
```

Comment:
the code is fine as it is, and i not holding an approval on this.
But you you want to step up the game of cleaning up, you can remove at least 2 ternaries with something like this:

```ts
let colorValue;
let opacityValue;
if (!value) { 
  colorValue = 'none';
} else if(...) {
  colorValue = `#SVGID_${value.id}`;
} else {
 ....
 do color value and opacity value;
}
if (separator === ':') {
  return `${prop}: ${colorValue} ${opacityValue ? `${prop}-opacity: ${opacity.toString()}; ` : ''}`
} else {
 return `${prop}="${color.toRgb()}" ${opacityValue ? `${prop}-opacity="${opacity.toString()}" ` : ''}`
}

```

this maybe is easier to read? just a suggestion follow your gut feeling is not a big deal